common-model/src/main/java/com/sequenceiq/common/api/telemetry/model/Telemetry.java

```diff
@@ -23,10 +24,10 @@
     private String databusEndpoint;
 
     @JsonProperty("meteringEnabled")
-    private boolean meteringEnabled;
+    private TelemetrySetting meteringEnabled;
 
     @JsonProperty("reportDeploymentLogs")
-    private boolean reportDeploymentLogs;
+    private TelemetrySetting reportDeploymentLogs;
```

Comment:
reportLogsEnable

Reply:
i do not like the format then: `reportDeploymentLogsEnable: "enabled"`, then it should be reportDeploymentLogsEnable: "true", so `reportDeploymentLogs : "enabled"` more meaningful as we want to use enable(d) as a value

Reply:
daemonLogs?

Reply:
that feature will look like that: report service logs after cluster install/failure after deployment (for like 20 mins, but nothing after that), so just reportLogs or daemonLogs would not be correct